validator/src/commands/contact_info/mod.rs

```diff
@@ -36,3 +52,43 @@ pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
         print!("{contact_info}");
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
If you're looking to be exhaustive, you're missing the case where you pass:
```rust
vec![COMMAND]
```
which yields:
```rust
            ContactInfoArgs {
                output: None
            },
```